src/CacheItemInterface.php

```diff
@@ -46,7 +46,7 @@ public function getKey();
      * @return mixed
      *   The value corresponding to this cache item's key, or null if not found.
      */
-    public function get();
+    public function get(): mixed;
```

Comment:
Sorry, for my ignorance, but does `mixed` include null?

Reply:
Yes, `null` is included:  https://www.php.net/manual/en/language.types.declarations.php#language.types.declarations.mixed